packages/next-swc/crates/core/src/next_ssg.rs

```diff
@@ -143,6 +143,21 @@ impl Fold for Analyzer<'_> {
         e
     }
 
+    fn fold_jsx_element(&mut self, jsx: JSXElement) -> JSXElement {
+        match &jsx.opening.name {
+            JSXElementName::Ident(i) => {
+                self.add_ref(i.to_id());
+            }
+            _ => {}
```

Comment:
I think the leftmost identifier of `JSXElementName::JSXMemberExpr` should be added as a ref.

Reply:
thanks, I've added the fix alongside the updated test. lmk what you think